eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -1,17 +1,26 @@
 parameters:
   rootFolder: '$(Build.SourcesDirectory)'
   runProxy: true
+  targetVersion: ''
 
 steps:
   - pwsh: |
         $(Build.SourcesDirectory)/eng/common/scripts/trust-proxy-certificate.ps1
     displayName: 'Language Specific Certificate Trust'
 
   - pwsh: |
+      $version = $(Get-Content C:\repo\sdk-for-python\eng\common\testproxy\target_version.txt -Raw).Trim()
```

Comment:
Is the parameter necessary since you're doing the lookup in the script?

Reply:
YEs. Allowing override.